core/mutations/ToggleFavourite.js

```diff
@@ -0,0 +1,7 @@
+import gql from "graphql-tag";
+
+export default gql`
+  mutation toggleFavourite($articleId: ID!) {
+    isFavourite(articleId: $articleId)
```

Comment:
Weird naming here 'isFavourite'.
Mutations name should represent imperative actions like `toggleFavourite` or `setFavourite`

Reply:
I used isFavourite since is the result of the mutation (whether an article is or isnt favourite is returned in a boolean), it used to be toggleFavourite but was changed favoring this, do you want it to be reversed to toggle?

Reply:
I'm torn about this mutation. I can't really see what the UI would to with this `boolean` value that doesn't require some logic on the client. 

I think a better API could be something like  `addToFavorites` that returns the resulting users favorites list like the `favorites` inside the User. So the UI gets updated accordingly.

Reply:
I'm going to request a re-review for this PR for the name updates, I have a TODO list with this change and the input ones for when I have access to the api as well